### PR TITLE
Adding chromatic in Travis

### PR DESCRIPTION
**Pull request by hakimelek**
Similar to #234 with Travis integration. 

Adding CHROMATIC_APPCODE as an env variable to travis.

### CODE REVIEW

**Comment by zol** [comment]
You may need to double check the `CHROMATIC_APPCODE` variable is exposed for public PRs. Not sure about Travis in particular but some CI systems won't pass env vars for builds started by non-trusted users for security reasons. If this is the case, you can safely commit the `appCode` into your code, it's ok for it to be publicly viewable!

**Comment by hakimelek** [comment]
Thanks for the quick review! I will add the `--exit-zero-on-changes`. I am assuming that the non-trusted users issue is not a problem for Travis, since we have other env variables already setup and never had any problems with the build when other users than owners for example push branches.

File: package.json [hunk 1]
```diff
@@ -78,6 +80,7 @@
     "setupTestFrameworkScriptFile": "./test-setup.js"
   },
   "dependencies": {
+    "babel-runtime": "^6.26.0",
```
Comment (jxnblk) [inline package.json]: You didn't mean to add this dependency, did you?

Comment (zol) [reply package.json]: It's not required for Chromatic.

Comment (hakimelek) [reply package.json]: Nope, good catch, that was part of the old branch.

**Comment by zol** [comment]
👍

**Pull request merged**